views/js/qtiCreator/editor/containerEditor.js

```diff
@@ -31,8 +31,10 @@ define([
     'taoQtiItem/qtiCreator/helper/creatorRenderer',
     'taoQtiItem/qtiCreator/helper/xincludeRenderer',
     'taoQtiItem/qtiCreator/editor/gridEditor/content',
-    'taoQtiItem/qtiCreator/editor/ckEditor/htmlEditor'
-], function(_, $, Promise, Loader, Container, Item, event, allQtiClasses, commonRenderer, xmlRenderer, simpleParser, creatorRenderer, xincludeRenderer, content, htmlEditor){
+    'taoQtiItem/qtiCreator/editor/ckEditor/htmlEditor',
+    'taoQtiItem/qtiCreator/editor/gridEditor/content',
+    'lib/dompurify/purify'
+], function(_, $, Promise, Loader, Container, Item, event, allQtiClasses, commonRenderer, xmlRenderer, simpleParser, creatorRenderer, xincludeRenderer, content, htmlEditor, contentHelper, DOMPurify){
```

Comment:
do we really need to add DOMPurify ?

Reply:
I think it's better to have clean in both sides, FE & BE, to give a fast error to user in case he tries to add something not supported like `<script>`

Reply:
What other sanitizers you considered for this? What is current approach for other parts of the app?

Reply:
> I think it's better to have clean in both sides, FE & BE, to give a fast error to user in case he tries to add something not supported like `<script>`

We also have escaping symbols and sanitazing on BE side: https://github.com/oat-sa/extension-tao-testqti/pull/2004

> What other sanitizers you considered for this? What is current approach for other parts of the app?
DOMPurify is already being used for example here: https://github.com/oat-sa/tao-core-ui-fe/blob/7835f8c7cbd7606bf4d9482e5bb357cc47f111f3/src/datatable.js#L892
There's also some escaping on BE, for other cases we can create separate tasks

Reply:
Indeed it should be sanitized both on FE and BE